R/generate_template.R

```diff
@@ -313,3 +345,53 @@ create_new_application <- function(name, location, sampleapp = FALSE, resetbutto
 
     return()
 }
+
+create_default_theme_file <- function(theme_file) {
+    lines <- c("### primary_color",
+               "# To set primary status color, that affect the color of header, valueBox, infoBox and box",
+               "# Valid possible value are colors names or hex-decimal value (i.e,: \"blue\", \"#086A87\")",
+               "# Blank/empty value will use default value",
+               "primary_color: \n\n",
+               
+               
+               "# Sidebar variables: change the default sidebar width, colors:",
```

Comment:
Can you recheck Connie's propsed names from the recording. Some of them are different from what connie mentioed in the recording .

Reply:
done